cdc/sink/codec/canal_flat_test.go

```diff
@@ -118,6 +118,12 @@ func (s *canalFlatSuite) TestNewCanalFlatMessage4DML(c *check.C) {
 			continue
 		}
 
+		// for `Column.Value` is nil, which mean's it is nullable, set the value to `""`
+		if obtainedValue == nil {
+			c.Assert(item.expectedValue, check.Equals, "")
```

Comment:
Will this empty value related to the column type？

Reply:
no, I think all column type can be `null`.